stories/core/Select.stories.tsx

```diff
@@ -302,3 +302,58 @@ Custom.args = {
     { value: 'red', label: 'Red' },
   ],
 };
+
+export const MiddleTextTruncation: Story<SelectProps<string>> = (args) => {
+  const {
+    options = [
+      {
+        value:
+          'MyFileWithAReallyLongNameThatWillBeTruncatedBecauseItIsReallyThatLongSoHardToBelieve_FinalVersion_V2.html',
+        label:
+          'MyFileWithAReallyLongNameThatWillBeTruncatedBecauseItIsReallyThatLongSoHardToBelieve_FinalVersion_V2.html',
+      },
+      { value: 'ShortNameFile.jpg', label: 'ShortNameFile.jpg' },
+      { value: 'SomeOtherFile.dgn', label: 'SomeOtherFile.dgn' },
+    ],
+    placeholder = 'Placeholder text',
+    ...rest
+  } = args;
+  const [selectedValue, setSelectedValue] = useState<string | undefined>(
```

Comment:
The selected value is not being updated when selecting a different option
![text truncation](https://user-images.githubusercontent.com/88331924/135679416-48b67f84-06aa-4f6b-a89b-2906cf6069b7.gif)

Reply:
Yeah, was issue with truncation component. Should be fixed